### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by ger619**
- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.

- You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.

- After that, you could implement the popup window.

### CODE REVIEW

File: portfolio.css [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+  width: 93px;
+  left: 135px;
+  top: 0;
+  border-radius: 8px;
+  padding: 4px 12px 4px 12px;
+}
+
+.modal-3 {
+  list-style: none;
+  background-color: #e5e5e5;
```
Comment (phelian23) [inline portfolio.css]: - [ ] Kindly make sure that the popup window is showing all the details of the projects. as it is, part of the project details are cut off, and the modal is not scrollable. kindly fix it.
Capture

File: portfolio.css [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+  }
+
+  .modal-1 {
+    color: #6070ff;
+    display: flex;
+    gap: 4px;
+    padding-bottom: 20px;
+    margin-top: 20px;
+    font-size: 12px;
+  }
```
Comment (phelian23) [inline portfolio.css:101]: - [ ] Kindly make sure the buttons in the popup matches that of the Figma template
Your design
Capture2
Figma Design
figma

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Linters are passing
✅ Descriptive pull request
✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.css (hunk 1 above)
Comment (ger619) [reply portfolio.css]: Screenshot 2022-04-07 at 23 33 47
This appears differently depending on the browser. Its browser compatibility

Comment (ger619) [reply portfolio.css]: @phelian23 I am unable to control it

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @ger619, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**